tests/probes_test.go

```diff
@@ -51,17 +51,10 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 		}
 		table.DescribeTable("should succeed", func(readinessProbe *v12.Probe, serverStarter func(vmi *v12.VirtualMachineInstance, port int, isFedoraVM bool)) {
 			// nc is used to create the HTTP server. The one shipped in CirrOS has a bug where it throws a
-			// 'connection reset by peer' when bound to IPv6 addresses. On those scenarios, we fallback to using a
-			// Fedora VM.
-			useFedoraVM := tests.IsIPv6Cluster(virtClient) && readinessProbe == httpProbe
+			// 'connection reset by peer' when bound to IPv6 addresses. Thus we use Fedora VM.
```

Comment:
I think that we were safe using cirros and that the bug was not related to cirros but to something at content-length, @maiqueb knos better.

Reply:
I'll eventually provide a decent explanation on https://github.com/kubevirt/kubevirt/pull/4129/commits/b41e635714a240fc082c8b7228ca5e10c8817d3b .

Reply:
dont we want to keep StartHTTPServer supporting the two VMs types ?
and if we dont, then the bool flag can be removed all over (i think i still saw it in couple places?)

Reply:
> dont we want to keep StartHTTPServer supporting the two VMs types ?

Do you have a reason to keep it ? I don't.

> and if we dont, then the bool flag can be removed all over (i think i still saw it in couple places?)
> https://github.com/kubevirt/kubevirt/pull/4151/files#diff-0836acab6741ba8251f478229fe81a6dR71

Have you seen the commit I pasted on https://github.com/kubevirt/kubevirt/pull/4151#discussion_r485528493 ? It removes that.

Reply:
> > dont we want to keep StartHTTPServer supporting the two VMs types ?
> 
> Do you have a reason to keep it ? I don't.
> 
> > and if we dont, then the bool flag can be removed all over (i think i still saw it in couple places?)
> > https://github.com/kubevirt/kubevirt/pull/4151/files#diff-0836acab6741ba8251f478229fe81a6dR71
> 
> Have you seen the commit I pasted on [#4151 (comment)](https://github.com/kubevirt/kubevirt/pull/4151#discussion_r485528493) ? It removes that.

Ye saw, it confused me with the code on this PR,
maybe worth also to write which PR this commit belongs to (less confusing and we can see easily when its merged and so on)